fedmsg.d/config.py

```diff
@@ -13,7 +13,9 @@
     'estuary_updater.enabled': True,
     'estuary_updater.log_level': logging.INFO,
     'estuary_updater.topics': [
-        '/queue/Consumer.client-estuary-updater.dev.VirtualTopic.eng.freshmaker.event.state.changed'
+        '/queue/Consumer.client-estuary-updater.dev.VirtualTopic.eng.' +
```

Comment:
The general way to do this is usually:
```python
('/queue/Consumer.client-estuary-updater.dev.VirtualTopic.eng.'
 'freshmaker.event.state.changed')
```

However, I think you should just put it in one line and add an entry in `.flake8` that excludes checking this file. You can see the docs on this at:
http://flake8.pycqa.org/en/latest/user/configuration.html